contracts/permissionless/PermissionlessOrderBookReserveLister.sol

```diff
@@ -0,0 +1,89 @@
+pragma solidity 0.4.18;
+
+
+import "./OrderBookReserve.sol";
+import "../FeeBurnerInterface.sol";
+
+
+contract KyberNetworkInterface {
```

Comment:
I don't think you should have it here.

Reply:
well
all the below are not defined in the interface.
I can add them there.
but will require changes in contract.
Since if interface is inherited the "automatic"  API for public parameters should be modified.